README.md

```diff
@@ -28,9 +28,14 @@ The above command, if it works correctly, will give you a link to view the gener
 
 ### Linting docs
 
-We use MarkdownLint to lint the markdown files. This runs as a test on GitHub actions every time you raise a pull request. You can also run this on your local environment either by using the [VSCode extension](https://marketplace.visualstudio.com/items?itemName=DavidAnson.vscode-markdownlint) or using the [Node package](https://github.com/DavidAnson/markdownlint) included. In the case of the latter, you can use `yarn` commands to run the linters.
+We use MarkdownLint to lint the markdown files. This runs as a test on GitHub actions every time you raise a pull request.
+
+#### Fixing linting issues
+
+You can also run this on your local environment either by using the [VSCode extension](https://marketplace.visualstudio.com/items?itemName=DavidAnson.vscode-markdownlint) or using the [Node package](https://github.com/DavidAnson/markdownlint) included. In the case of the latter, you can use `yarn` commands to run the linters.
```

Comment:
@Kalaiselvan88, breaking it down like this has removed the lint commands from the above section.

The above section should have the code block as is. The new heading will have the two commands related to fixing linting issues:
```
yarn lint-fix:md
yarn lint-fix:prettier
```

Even the extension and other notes are relevant to the above section. This section can just be something like this:

> If you are seeing linting errors, most of them can be fixed by running scripts as below. It's a good idea to run the lint commands again to check if some of the errors couldn't be fixed and will need to be fixed manually.

Reply:
Updated. Thank you.